DQM/HcalTasks/plugins/QIE10Task.cc

```diff
@@ -33,164 +33,124 @@ QIE10Task::QIE10Task(edm::ParameterSet const& ps):
 	edm::ESHandle<HcalDbService> dbs;
 	es.get<HcalDbRecord>().get(dbs);
 	_emap = dbs->getHcalMapping();
-	std::vector<uint32_t> vhashC36;
-	vhashC36.push_back(HcalElectronicsId(36, 3,
-		FIBER_uTCA_MIN1, FIBERCH_MIN, false).rawId());
-	_filter_C36.initialize(filter::fPreserver, hcaldqm::hashfunctions::fCrateSlot,
-		vhashC36);
 
-	//	create a Dual Anode filter
-	std::vector<uint32_t> vhashDA;
-	for (int depth=1; depth<=2; depth++)
-		for (int ieta=29; ieta<=41; ieta++)
-			vhashDA.push_back(HcalDetId(HcalForward, ieta, 39, depth));
-	_filter_DA.initialize(filter::fPreserver, hcaldqm::hashfunctions::fDChannel,
-		vhashDA);
+	unsigned int nTS = _ptype==fLocal ? 10 : 6;
+
+	// create a slot filter and initialize what you need
+	unsigned int itr=0;
+	for(unsigned int crate=22; crate < 33; ++crate) {
```

Comment:
OK, I added a vector constants::crateListHF to Constants.h, and use this for these loops instead.